charts/sck-otel/templates/_config.tpl

```diff
@@ -293,6 +293,14 @@ processors:
     - key: service.name
       from_attribute: k8s.pod.labels.app
       action: upsert
+  filter/log:
+    logs:
+      # any logs matching filters are excluded from remainder of pipeline
+      exclude:
+        match_type: strict
+        resource_attributes:
+          - key: com.splunk.exclude
```

Comment:
Is `com.splunk.exclude` attribute set anywhere in the pipeline before? Shouldn't it be `k8s.pod.annotations.splunk.com/exclude` and `k8s.namespace.annotations.splunk.com/exclude` instead?